public_html/documentation.php

```diff
@@ -98,6 +101,74 @@ function build_sidebar_nav($elements) {
     '<p class="small text-end mt-3 d-none d-md-block"><a href="#" class="text-muted"><i class="fas fa-arrow-to-top"></i> Back to top</a></p>';
 $toc_nav .= '</nav>';
 
+$md_content_replace[] = ['<!-- usage_toc -->'];
+if (in_array($_GET['path'], ['docs'])) {
+    $inline_toc = '<div class="row"><div class="col-6 col-md-3"><div class="mb-3 h3">Usage</div>';
+    foreach ($sidebar_nav_elements['usage'] as $mdfile => $mdcontent) {
+        if (isset($mdcontent['url'])) {
+            $mdcontent['url'] = str_replace('docs/', '', $mdcontent['url']);
+            $inline_toc .=
+                '<div class="row">' .
+                '<a href="' .
+                $mdcontent['url'] . '">' . $mdcontent['title'] .
+                '</div>';
+        }
+        else {
+            $mdcontent[0]['url'] = str_replace('docs/', '', $mdcontent[0]['url']);
+            $inline_toc .=
+                '<div class="row"><div class="ml-1">' .
```

Comment:
```suggestion
                '<div class="row"><div class="ms-1">' .
```
Since bootstrap 5 you have to use `start` and `end` instead of `left` and `right`